### PR TITLE
INC New API TF oob, wide_deep_large_ds, 3dunet-mlperf examples

### PR DESCRIPTION
**Pull request by lvliang-intel**
Signed-off-by: Lv, Liang1 <liang1.lv@intel.com>

## Type of Change

feature
API not changed

## Description

detail description 
JIRA ticket: ILITV-2455

## Expected Behavior & Potential Risk

INC New API TF enable oob, wide_deep_large_ds, 3dunet-mlperf examples

## How has this PR been tested?

UT, Pre-CI and OOB extension test.

## Dependency Change?

No.

### CODE REVIEW

File: examples/tensorflow/oob_models/quantization/ptq/README.md [hunk 1]
```diff
@@ -86,8 +88,24 @@ List models names can get with open_model_zoo:
...
 
-## 5. Config the yaml file
-In examples directory, there is a config.yaml for tuning the model on Intel CPUs. The 'framework' in the yaml is set to 'tensorflow'. If running this example on Intel GPUs, the 'framework' should be set to 'tensorflow_itex' and the device in yaml file should be set to 'gpu'. The config_itex.yaml is prepared for the GPU case. We could remove most of items and only keep mandatory item for tuning. We also implement a calibration dataloader and have evaluation field for creation of evaluation function at internal neural_compressor.
+## 5. Quantization Config
+The Quantization Config class has default parameters setting for running on Intel CPUs. If running this example on Intel GPUs, the 'backend' parameter should be set to 'tensorflow_itex' and the 'device' parameter should be set to 'gpu'.
+
+```
+config = PostTrainingQuantConfig(
+    device="gpu",
+    backend="tensorflow_itex",
```
Comment (ftian1) [inline examples/tensorflow/oob_models/quantization/ptq/README.md:97]: who will clean this field to match latest definition @mengniwang95 ?

File: examples/tensorflow/oob_models/quantization/ptq/tf_benchmark.py [hunk 2]
```diff
@@ -330,29 +329,62 @@ def __iter__(self):
...
-        quantizer = Quantization("./config_tmp.yaml")
+        from neural_compressor.experimental import common
+        from neural_compressor.quantization import fit
+        from neural_compressor.config import PostTrainingQuantConfig, \
+            TuningCriterion, AccuracyCriterion, AccuracyLoss, set_random_seed
+
+        set_random_seed(9527)
+
+        tuning_criterion = TuningCriterion(
+            strategy="basic",
```
Comment (ftian1) [inline examples/tensorflow/oob_models/quantization/ptq/tf_benchmark.py:343]: for default value, we shouldn't list it in code

File: examples/tensorflow/oob_models/quantization/ptq/tf_benchmark.py [hunk 3]
```diff
@@ -330,29 +329,62 @@ def __iter__(self):
...
+            timeout=0,
+            max_trials=100,
+            objective="performance")
+
+        tolerable_loss = AccuracyLoss(loss=0.01)
```
Comment (ftian1) [inline examples/tensorflow/oob_models/quantization/ptq/tf_benchmark.py:348]: remove AccuracyLoss class

**Comment by hshen14** [comment]
@chensuyue , where can we see the extension test report?

**Comment by chensuyue** [comment]
> @chensuyue , where can we see the extension test report?

This PR is not ready for extension test, still modifying.

**Comment by ftian1** [comment]
@chensuyue  who will remove those old examples from INC master for 2.0 release?

**Comment by chensuyue** [comment]
> @chensuyue who will remove those old examples from INC master for 2.0 release?

I will do that.

File: examples/tensorflow/oob_models/quantization/ptq/tf_benchmark.py [hunk 4]
```diff
@@ -330,29 +329,38 @@ def __iter__(self):
     # tune
     if args.tune:
         # os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
-        from neural_compressor.experimental import Quantization, common
         inputs = model_detail['input']
         outputs = model_detail['output']
-        _write_inputs_outputs_to_yaml(args.yaml, "./config_tmp.yaml", list(inputs.keys()), outputs)
 
-        quantizer = Quantization("./config_tmp.yaml")
+        from neural_compressor.experimental import common
```
Comment (ftian1) [inline examples/tensorflow/oob_models/quantization/ptq/tf_benchmark.py:335]: this usage is incorrect

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3719/artifact/report.html) pls check perf gap of wide_deep_large_ds.

**Comment by chensuyue** [comment]
OOB model with multi-instance, [test](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3788/artifact/report.html)

**Comment by chensuyue** [comment]
retest [wide_deep_large_ds](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3800/)

**Comment by chensuyue** [comment]
retest [wide_deep_large_ds](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3808/)

**Comment by chensuyue** [comment]
test [3dunet-mlperf](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3810/)

**Comment by chensuyue** [comment]
> retest [wide_deep_large_ds](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3808/)

WnD test pass.

**Comment by chensuyue** [comment]
> test [3dunet-mlperf](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3810/)

need to fix the bug.

**Comment by chensuyue** [comment]
> > test [3dunet-mlperf](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3810/)
> 
> need to fix the bug.

Failed due to read only issue, will fix later.

**Review by chensuyue — APPROVED** [review]
